python/scripts/prepare_fineweb_edu.py

```
@@ -0,0 +1,290 @@
+    total = len(train_input) + len(val_input)
+    print(f"\n{'=' * 60}")
+    print("FineWeb-Edu data preparation complete")
+    print(f"{'=' * 60}")
+    print(f"  Output:        {out_dir}")
+    print(f"  Source:        FineWeb-Edu score >= {args.min_score}")
+    print(f"  Vocab:         {actual_vocab:,}")
+    print(f"  Total tokens:  {total:,}")
+    print(f"  Train tokens:  {len(train_input):,}")
+    print(f"  Val tokens:    {len(val_input):,}")
+    print(f"  Documents:     {len(train_docs) + len(val_docs):,}")
+    print(f"  Mask ratio:    0.0% (all tokens are valid targets)")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)